invenio_app_ils/circulation/mail/tasks.py

```diff
@@ -10,11 +10,25 @@
 from celery import shared_task
 from celery.utils.log import get_task_logger
 from flask import current_app
+from invenio_circulation.proxies import current_circulation
 from invenio_mail.tasks import send_email
 
+from invenio_app_ils.api import Document
+from invenio_app_ils.circulation.mail.factory import loan_message_factory, \
+    overdue_loan_message_factory
+from invenio_app_ils.circulation.utils import circulation_overdue_loan_days
+from invenio_app_ils.records.api import Patron
+
 celery_logger = get_task_logger(__name__)
 
 
+def get_recipients(patrons):
+    """Return test recipients when ENABLE_TEST_RECIPIENTS is True."""
+    if current_app.config["ENABLE_TEST_RECIPIENTS"]:
+        return current_app.config["MAIL_NOTIFY_TEST_RECIPIENTS"]
```

Comment:
@zzacharo We don't need fake mails, in the `config.py` the variable `MAIL_SUPPRESS_SEND` instead of sending actual mails, just logs them in the console. You cannot get more chilled than that :)
```
#: Disable email sending by default.
MAIL_SUPPRESS_SEND = True
```

The test recipients are for testing if we can actually send emails.

@kprzerwa I used the inveniosoftware email so we override it in the cds layer with the cds related stuff ;)